ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -447,7 +447,18 @@ objects:
           done < /tmp/containers
 
           echo "Gathering kube-apiserver audit.log ..."
-          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/audit.log
+          oc adm node-logs --role=master --path=kube-apiserver/ > /tmp/kube-audit-logs
+          while IFS=$'\n' read -r line; do
+            IFS=' ' read -ra log <<< "${line}"
+            FILTER=gzip queue /tmp/artifacts/nodes/"${log[0]}"-"${log[1]}".gz oc adm node-logs "${log[0]}" --path=kube-apiserver/"${log[1]}"
+          done < /tmp/kube-audit-logs
```

Comment:
@smarterclayton ptal now? the only downside of this approach is that we're getting each audit file in a separate file, not sure if the current `queue` implementation handles writing to it several times, but I haven't seen that

Reply:
I wouldn't try.  Probably would be too much work.